models/public/anti-spoof-mn3/accuracy-check.yml

```diff
@@ -0,0 +1,29 @@
+models:
+  - name: anti-spoofing-mn3
+
+    launchers:
+      - framework: dlsdk
+        adapter: classification
+        tags:
+          - FP32
+        model: anti-spoof-mn3.xml
+        weights: anti-spoof-mn3.bin
```

Comment:
please remove tag and model paths, we use command line args for getting model path